src/scripts/clipperUI/components/pdfPageSelectionRadioButton.tsx

```diff
@@ -60,9 +60,7 @@ class PdfPageSelectionRadioButton extends ComponentBase<{}, ClipperStateProp> {
 			shouldShowPopover: false
 		} as PdfPreviewInfo), this.props.clipperState.setState);
 
-		if (!selection) {
-			document.getElementById(Constants.Ids.rangeInput).focus();
-		}
+		selection ? document.getElementById(Constants.Ids.radioAllPagesLabel).focus() : document.getElementById(Constants.Ids.rangeInput).focus();
```

Comment:
This syntax is shorter: `document.getElementById(selection ? Constants.Ids.radioAllPagesLabel : Constants.Ids.rangeInput).focus()